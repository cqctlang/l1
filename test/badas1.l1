badas = mkas([:]);
x = (int *){badas}0;
*x;
