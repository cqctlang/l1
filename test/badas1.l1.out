error: incompletely defined address space
	#0                    mkas	(builtin function)
	#1                   entry	(badas1.l1:1)
